dashboard/src/app/index.module.ts

```diff
@@ -72,7 +72,13 @@ function keycloakLoad(keycloakSettings: any) {
     script.async = true;
     script.src = keycloakSettings['che.keycloak.js_adapter_url'];
     script.addEventListener('load', resolve);
-    script.addEventListener('error', () => reject('Error loading script.'));
+    script.addEventListener('error', () => {
+      return reject(`<h3><b>Unknown Error</b></h3>
+ <span><p>Probably, one of Che hosts is signed with a self-signed certificate. Possible solutions would be:</p>
+ <p>1. Import CA into your browser, you can find instruction how to do it by 
+ <a href="${DEFAULT_DOCS_CERTIFICATE}" target="_blank">documentation</a>.</p>
```

Comment:
Is there any reason not to use `cheBranding.getDocs().certificate` in favour of `DEFAULT_DOCS_CERTIFICATE`?

Reply:
This message shows before bootstrapping. We do not have
a cheBranding object in the case with this error. So, I am using DEFAULT_DOCS.

Reply:
Got it, thanks.

But I see that you updated service `cheBranding` so it can handle link to documentation for certificate issues in `product.json`. This means that and if such issue happens then default link could be shown instead of the correct one provided as `docs.certificate` field.

WDYT of getting `product.json` using XHR to get correct documentation link?

Reply:
Done(I have completely reworked branding).